collect-adb.rst

```diff
@@ -0,0 +1,108 @@
+*****************************************
+Using Android Debug Bridge with Collect
+*****************************************
+
+`Android Debug Bridge <https://developer.android.com/studio/command-line/adb.html>`_ is a command which acts as a bridge between the Android device and the terminal. It can control device over USB from a computer, copy files back and forth, install and uninstall apps, run shell commands etc. For the developers and users of ODK Collect, the most common uses are:
+
+- loading blank forms to SD Card
+- fetching completed forms
+- deleting forms
+- copying the form database
+- making the **.apk** file from the source code
```

Comment:
installing the apk, right? adb doesn't make the apk....

Reply:
I searched for this. It can build as well as install it

Reply:
really?
maybe that's worth a note in the relevant section of this doc

Reply:
However in the project's context "installing" makes more sense